autometa/binning/recursive_dbscan.py

```diff
@@ -546,7 +649,9 @@ def binning(
     markers: pd.DataFrame,
     domain: str = "bacteria",
     completeness: float = 20.0,
-    purity: float = 90.0,
+    purity: float = 95.0,
+    coverage_stddev: float = 25.0,
+    gc_content_stddev: float = 5.0,
```

Comment:
These defaults are I think defined twice in the file. Is there a way of making these just defined in one place? I am thinking that having two definitions allows for the possibility of errors when others are editing the file and not realizing the double definition.

Reply:
We can make them required arguments for most areas and have the default in just one place

Reply:
These defaults have been removed from `get_clusters(...)`. Now the only default is placed in `def binning(...)`. We can also remove the defaults from this if you would only like the defaults in the argparse parameters.